packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -111,8 +111,6 @@ import ovhManagerCloudConnect from '@ovh-ux/manager-cloud-connect';
 import { detach as detachPreloader } from '@ovh-ux/manager-preloader';
 import ovhNotificationsSidebar from '@ovh-ux/manager-notifications-sidebar';
 import ovhManagerAccountMigration from '@ovh-ux/manager-account-migration';
-import { pollingService } from '@ovh-ux/manager-bm-server-components';
```

Comment:
it doesn't seem it should be part of this PR right ?

Reply:
commit has been dropped